python/samples/04.ai.f.dataSource.azureOpenAI/appPackage/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+    "version": "1.0.0",
+    "manifestVersion": "1.16",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.package.name",
+    "name": {
+        "short": "TwentyQuestions-${{TEAMSFX_ENV}}",
```

Comment:
file requires a few updates to match this sample

Reply:
manifest updated.